src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java

```diff
@@ -131,7 +136,17 @@ public void drive(double x, double y, double rot) {
     
     // Second method passing to moduleStates
     public void drive(ChassisSpeeds speeds){
-        SwerveModuleState[] moduleStates = kinematics.toSwerveModuleStates(speeds);
+        ChassisSpeeds newSpeeds = speeds;
+        if (robotContainer.getAuto()) {
+            newSpeeds = speeds.minus(new ChassisSpeeds(speeds.vxMetersPerSecond*2, 0, 0));
```

Comment:
What's the point of this? This will drive it in reverse? (vx - 2*vx = -vx)

Reply:
https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/DriverStation.html#isAutonomous()

Reply:
Yes this was my janky solution to reverse auto on only one axis 😅 And thanks for the command I wasnt sure how to do that